ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewManager.java

```diff
@@ -64,6 +65,11 @@ public void setBorderStyle(ReactViewGroup view, @Nullable String borderStyle) {
     view.setBorderStyle(borderStyle);
   }
 
+  @ReactProp(name = "elevation")
+  public void setElevation(ReactViewGroup view, float elevation) {
+    ViewCompat.setElevation(view, PixelUtil.toPixelFromDIP(elevation));
```

Comment:
I suspect that's essentially what `ViewCompat.setElevation` does, with the addition that if Android ever does backport `elevation` to earlier APIs (admittedly unlikely) it'll start working there as well. But yeah, doing it the other way would work fine too. Is there a reason to prefer one style over the other?